modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -1011,7 +1015,7 @@ def test_merge_preserves_metadata(has_cols_metadata, has_dtypes_metadata):
 
     if has_dtypes_metadata:
         # Verify that there were initially materialized metadata
-        assert modin_frame.has_dtypes_cache
+        assert modin_frame.has_materialized_dtypes
```

Comment:
we now have some dtypes cache quite often, so the old check no longer works correctly, what it wants to check is whether we have materialized dtypes